### PR TITLE
add overview

### PR DESCRIPTION
**Pull request by phoebemirman**
https://github.com/AugurProject/augur/issues/1172

### CODE REVIEW

**Comment by AugurIntegration** [comment]
This pull request **introduces 1 alert** and **fixes 1** when merging 3532085f7a09c419721bd4006520c659288d0894 into f43dfff0f705ac47888e182150c67d5cc781423d - [view on LGTM.com](https://lgtm.com/projects/g/AugurProject/augur-ui/rev/pr-7a1c2ef678c34262b2726df96ad787a5f5ee162b)

**new alerts:**

* 1 for Unused variable, import, function or class

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: src/modules/common-elements/constants.ts [hunk 1]
```diff
@@ -603,3 +603,20 @@ export const NOTIFICATION_TYPES = {
...
 };
+
+// Account View - Timeframe selection options 
+// # Timeframe readable names
+export const TIMEFRAMES = {
+  HOUR: "24 hr",
+  DAY: "1 Week",
+  WEEK: "1 Month",
+  ALL: "All Time",
+};
```
Comment (bthaile) [inline src/modules/common-elements/constants.ts]: I got confused a bit on constants names. DAY: “1 Week”

How about DAY: 24hr, ...

**Comment by AugurIntegration** [comment]
This pull request **introduces 1 alert** and **fixes 1** when merging 68c2ca3687e076f2c2a1c301ed4c5cbd91c705a3 into f43dfff0f705ac47888e182150c67d5cc781423d - [view on LGTM.com](https://lgtm.com/projects/g/AugurProject/augur-ui/rev/pr-6df8fbeb47500fd2645a4eb85b3436d2d2689842)

**new alerts:**

* 1 for Unused variable, import, function or class

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: src/modules/common-elements/constants.ts (hunk 1 above)
Comment (phoebemirman) [reply src/modules/common-elements/constants.ts]: yeah. that was a mixup

**Comment by AugurIntegration** [comment]
This pull request **introduces 1 alert** and **fixes 1** when merging c04b3d538c76891ebf36e805ee8bc53d9879a8f9 into f43dfff0f705ac47888e182150c67d5cc781423d - [view on LGTM.com](https://lgtm.com/projects/g/AugurProject/augur-ui/rev/pr-e96eb3c5c0eff486509643abecd1a4e31da3e6dd)

**new alerts:**

* 1 for Unused variable, import, function or class

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request merged**